flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/calcite/FlinkSchemaVersion.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.calcite;
+
+import org.apache.calcite.schema.SchemaVersion;
+
+/** Schema version for time travel. */
+public abstract class FlinkSchemaVersion implements SchemaVersion {
```

Comment:
I'm wondering why we need  `FlinkSchemaVersion` since we already had `TimestampSchemaVersion`?
Can't we use `TimestampSchemaVersion` directly?

Reply:
Yes we can.